### PR TITLE
Feat/notifcation

### PR DESCRIPTION
**Pull request by sachins-geekyants**
ticket :- 
https://github.com/atsign-foundation/apps/issues/804
https://github.com/atsign-foundation/apps/issues/806
[Figma](https://www.figma.com/file/nNys7N6irbXsvM2SbAEgl8/AtmospherePro-Desktop-V2?type=design&node-id=0-1&mode=design) design

**- What I did**
- Added notification dialog to show recent notifications

**- How I did it**
- Added notification icon on header
- when icon is pressed, opening the notification dialog
- based on different notifications, there are different status card

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by sonle-geekyants** [comment]
@sachins-geekyants Please resolve the remaining issues: [#304](https://github.com/atsign-foundation/atmosphere_pro/issues/344)

**Comment by sachins-geekyants** [comment]
@sonle-geekyants resolved the issue

File: lib/desktop_screens_new/notification/sending_file_card.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                  width: (Platform.isAndroid || Platform.isIOS) ? 200 : 230,
+                  child: Text(
+                    getAtsignCountFromFileTransfer(fileTransfer!),
+                    style: TextStyle(fontSize: 9, fontWeight: FontWeight.bold),
+                  ),
+                ),
+                SizedBox(height: 5),
+                LinearProgressIndicator(
+                  minHeight: 10,
+                  borderRadius: BorderRadius.circular(5),
```
Comment (sonle-geekyants) [inline lib/desktop_screens_new/notification/sending_file_card.dart]: @sachins-geekyants  `Error: No named parameter with the name 'borderRadius'`

**Review by sonle-geekyants — CHANGES_REQUESTED** [review]

File: lib/desktop_screens_new/notification/sending_file_card.dart (hunk 1 above)
Comment (sachins-geekyants) [reply lib/desktop_screens_new/notification/sending_file_card.dart]: I can run it
Screenshot 2023-10-26 at 4 44 52 PM

**Comment by sonle-geekyants** [comment]
> @sonle-geekyants resolved the issue

@sachins-geekyants Added some new issues in the ticket!

**Review by sonle-geekyants — APPROVED** [review]

**Pull request merged**